Source/WebCore/dom/EventLoop.h

```diff
@@ -98,6 +98,8 @@ enum class HasReachedMaxNestingLevel : bool { No, Yes };
 // https://html.spec.whatwg.org/multipage/webappapis.html#event-loop
 class EventLoop : public RefCountedAndCanMakeWeakPtr<EventLoop> {
 public:
+    using Tasks = Vector<std::unique_ptr<EventLoopTask>, 4>;
```

Comment:
I would have called this either TaskVector or TaskList to be more explicit about the type.
How did we arrive at 4 as the optimal inline capacity?  Did we gather any stats on popular websites?

Reply:
For JS2 / SP3, small numbers are typically within 1-4. After that, it becomes much larger (like, 100), so 4 would be a good capacity.

Reply:
I see. Makes sense.